bench/tx-generator/src/Cardano/Benchmarking/Script/Env.hs

```diff
@@ -42,10 +44,12 @@ import           Cardano.Benchmarking.Script.Store
 
 import           Cardano.TxGenerator.Types (TxGenError(..))
 
-data Env = Env { dmap :: DMap Store Identity }
+data Env = Env { dmap :: DMap Store Identity
+               , protoParams :: Maybe ProtocolParameterMode
+               }
 
 emptyEnv :: Env
-emptyEnv = Env { dmap = DMap.empty }
+emptyEnv = Env { dmap = DMap.empty, protoParams = Nothing }
```

Comment:
Just a very minor nitpick about formatting -- I'd:

0. break the line, between `=` and `Env` aligning the latter naturally.
1. break the line, just before the `,`  aligning the latter with the `{`
2. make the trailing `}` on its own line

...which is how multi-line is done frequently.